src/gluonts/nursery/duru/vdvae_LICENSE

```diff
@@ -0,0 +1,9 @@
+License of [VDVAE](https://github.com/openai/vdvae/blob/main/LICENSE.md).
```

Comment:
Do we need to include this file? Can't we just link to it somewhere?

Reply:
Yes, we are using VDVAE's code partly. I'm referring to this also in the appendix in the functions which we are using.

Reply:
The MIT license of the repo states: 

"
Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the “Software”), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

*The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.*
"

Hence, I suggest to leave it in.

Reply:
Can we include the license in the head of the files where it applies and delete the separate file?